engine/server/sv_log.c

```diff
@@ -169,11 +169,21 @@ void SV_SetLogAddress_f( void )
 	int port;
 	string addr;
 
+	if( svs.log.net_log && Cmd_Argc() == 2 && !Q_strcmp( Cmd_Argv( 1 ), "off" )) 
+	{
+		svs.log.net_log = false;
+		memset( &svs.log.net_address, 0, sizeof( netadr_t ));
+
+		Con_Printf( "logaddress: disabled.\n" );
+
+		return;
+	}
+
 	if( Cmd_Argc() != 3 )
 	{
-		Con_Printf( "logaddress: usage\nlogaddress ip port\n" );
+		Con_Printf( "logaddress: usage\nlogaddress < ip port | off >\n" );
```

Comment:
While you're at it, replace usage line by more simple and common: `S_USAGE "logaddress < IP port | off >"`

Other than this, I'm fine with these changes.